code/src/containers/order-item/OrderItem.tsx

```diff
@@ -45,10 +46,13 @@ const OrderItem = ({ order }: OrderItemProps) => {
   );
 
   return (
-    <AppAccordionContainer className="spa-order-item__container">
+    <AppAccordionContainer
+      expanded={isExpanded}
+      className="spa-order-item__container"
+    >
       <AppAccordionSummary
         className="spa-order-item__title"
-        expandIcon={<ExpandMoreIcon />}
+        expandIcon={!isExpanded && <ExpandMoreIcon />}
```

Comment:
Why do we need to add expanded icon only when item is not expanded?

Reply:
I added a prop because we have this icon by default. On this page, we always need an expanded accordion without the possibility of changing it, so do not confuse users I just added it. 
![image](https://github.com/user-attachments/assets/57dcdf80-c1e6-497f-831f-a79b5b1dc7f6)